pom.xml

```diff
@@ -113,6 +112,7 @@
         <dependency>
             <groupId>log4j</groupId>
             <artifactId>log4j</artifactId>
+            <version>1.2.17</version>
```

Comment:
The `Log4JLog` class in the compile classpath needs it.  So it's `optional` instead of `test`.